src/cypress/integration/Form/TextArea/textarea-disabled.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { textareaCopy } from 'shared/data/copyContent';
+
+describe('TextArea Disabled tests', () => {
+  before(() => {
+    cy.visitStorybook();
+    cy.loadStory('components-form-textarea', 'disabled');
+  });
+
+  it('Should be displayed', () => {
+    cy.findByText(textareaCopy.disabled);
```

Comment:
In general, we want to be using `findByLabel` so there's two things we can do here:

1. Update the `TextArea` component to use a `label` instead of a `span` in this PR
2. Open an issue to refactor the `TextArea` to use a `label` (like what we did with `Checkbox`)

I'll leave it up to you 👍